qa/suites/fs/mirror-ha/overrides/whitelist_health.yaml

```diff
@@ -1,14 +0,0 @@
-overrides:
-  ceph:
-    log-ignorelist:
-      - overall HEALTH_
-      - \(FS_DEGRADED\)
-      - \(MDS_FAILED\)
-      - \(MDS_DEGRADED\)
-      - \(FS_WITH_FAILED_MDS\)
-      - \(MDS_DAMAGE\)
-      - \(MDS_ALL_DOWN\)
-      - \(MDS_UP_LESS_THAN_MAX\)
-      - \(FS_INLINE_DATA_DEPRECATED\)
-      - Reduced data availability
-      - Degraded data redundancy
```

Comment:
What about these two items, which are not in `qa/cephfs/overrides/ignorelist_health.yaml` ?

Reply:
I don't see these warnings in my latest run, so they are likely unnecessary.